website/cue/reference/remap/functions/find.cue

```diff
@@ -0,0 +1,63 @@
+package metadata
+
+remap: functions: find: {
+	category: "String"
+	description: """
+		Determines whether the elements in the `value` matches the `pattern` and returns its position or -1.
```

Comment:
This would technically be the "start of it's position"?

`foobar` with pattern = "bar" returns 3 (the index of "b")

Reply:
And is this being updated to error rather than -1?